db-resources/MongoDB/index.js

```diff
@@ -0,0 +1,93 @@
+// Connecting to MongoDB without Mongoose ORM
+const { MongoClient } = require('mongodb'), format = require('util').format;
+const { getRandomArbitrary, getRandomWhole, scoreRound, getRandomPcnt, generateRandomCountry, getProductId } = require('./seedHelper')
+const { performance } = require('perf_hooks')
+
+MongoClient.connect('mongodb://127.0.0.1:27017', { useNewUrlParser: true }, (err, client) => {
+  if (err) {
+    console.log('error', err)
+    client.close();
+  } else {
```

Comment:
No need to call close here.  On failure, you never connected in the first place

Reply:
Deleted!